src/Connections/GenericHook.ts

```diff
@@ -162,14 +163,12 @@ export class GenericHookConnection extends BaseConnection implements IConnection
         if (!config.generic) {
             throw Error('Generic Webhooks are not configured');
         }
-        const hookId = randomUUID();
-        const validState = GenericHookConnection.validateState(data, config.generic.allowJsTransformationFunctions || false);
-        await GenericHookConnection.ensureRoomAccountData(roomId, intent, hookId, validState.name);
-        await intent.underlyingClient.sendStateEvent(roomId, this.CanonicalEventType, validState.name, validState);
-        const connection = new GenericHookConnection(roomId, validState, hookId, validState.name, messageClient, config.generic, as, intent);
+        const connection = new GenericHookConnection(roomId, data, randomUUID(), messageClient, config.generic, as, intent);
```

Comment:
We've stopped validating the state here?

Reply:
Ditto my previous comment

Reply:
Reverted